packages/taiko-client/pkg/rpc/methods.go

```diff
@@ -322,6 +322,11 @@ func (c *Client) CalculateBaseFee(
 				return nil, fmt.Errorf("failed to fetch parent gas excess: %w", err)
 			}
 		}
+		// The time of l1Head should always be greater than that of l2Head.
+		// Since the block.Time after anvil rollback is smaller than the original one, the timestamp has to be modified here.
+		if currentTimestamp < l2Head.Time {
```

Comment:
Still think changing the code here is probly not the best way to resolve this test issue, let me figure out later.